### DISCUSSION TITLE
GitHub action 从image builder编译某些固件会失败，无法生成rootfs.tar.gz，有完整日志

### DISCUSSION THREAD

**Discussion by 14790897** [immortalwrt/immortalwrt#1502] [Q&A] [answered]
https://github.com/14790897/OpenWrt-In-Docker/actions/runs/11273465399/job/31350550934

**Comment by 1715173329** [comment]
为什么你会觉得所有 target 都会有 rootfs tarball 呢？

**Reply by 14790897** [reply]
我是新手，不晓得为什么

**Reply by 14790897** [reply]
> 为什么你会觉得所有 target 都会有 rootfs tarball 呢？

# CONFIG_TARGET_ROOTFS_TARGZ is not set
只要把这个改成yes就行了吗？

**Comment by 14790897** [comment, marked as answer]
.config文件中将# CONFIG_TARGET_ROOTFS_TARGZ is not set
改为CONFIG_TARGET_ROOTFS_TARGZ=y

**Reply by 14790897** [reply]
附带sh指令
```sh
          sed -i '/# CONFIG_TARGET_ROOTFS_TARGZ is not set/s/.*/CONFIG_TARGET_ROOTFS_TARGZ=y/' .config

```
